### ISSUE TITLE
bug: Different path on filtered result.

### ISSUE THREAD

**Issue by JCQuintas** [mrverdant13/coverde#50]
The result of the filter command is changing the path to an absolute one, which in turn makes the `coverde report` command to fail with the error below.

**lcov.info**

> SF:lib/cubit/dashboard/dashboard_cubit.dart
  
**filtered.lcov.info**

> SF:/Users/jcquintas/learner_app_flutter/lib/cubit/dashboard/dashboard_cubit.dart

**Error**
```stack
FileSystemException: Cannot open file, path = 'Users/jcquintas/learner_app_flutter/lib/cubit/dashboard/dashboard_cubit.dart' (OS Error: No such file or directory, errno = 2)
#0      _File.throwIfError (dart:io/file_impl.dart:635:7)
#1      _File.openSync (dart:io/file_impl.dart:479:5)
#2      _File.readAsBytesSync (dart:io/file_impl.dart:539:18)
#3      _File.readAsStringSync (dart:io/file_impl.dart:584:18)
#4      _File.readAsLinesSync (dart:io/file_impl.dart:590:36)
#5      CovFile.generateSubReport (package:coverde/src/entities/cov_file.dart:179:34)
#6      CovDir.generateSubReport (package:coverde/src/entities/cov_dir.dart:270:17)
#7      CovDir.generateSubReport (package:coverde/src/entities/cov_dir.dart:270:17)
#8      CovDir.generateSubReport (package:coverde/src/entities/cov_dir.dart:270:17)
#9      CovDir.generateReport (package:coverde/src/entities/cov_dir.dart:190:7)
#10     ReportCommand.run (package:coverde/src/commands/report/report.dart:169:9)
#11     CommandRunner.runCommand (package:args/command_runner.dart:209:27)
#12     CommandRunner.run.<anonymous closure> (package:args/command_runner.dart:119:25)
#13     new Future.sync (dart:async/future.dart:301:31)
#14     CommandRunner.run (package:args/command_runner.dart:119:14)
#15     coverde (package:coverde/src/commands/coverde.dart:24:16)
#16     main (file:///Users/jcquintas/.pub-cache/hosted/pub.dartlang.org/coverde-0.1.0+1/bin/coverde.dart:10:18)
#17     _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:32)
#18     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:192:12)
```

## Commands

```bash
flutter test --coverage
coverde filter -f '\.g.dart,\.freezed.dart' -m w
coverde report -i filtered.lcov.info
```

**Comment by mrverdant13** [comment]
Hey @JCQuintas 
Apologies for the delay with this issue
It's been fixed in the main branch, which you could use until I publish a new version is the package.
Really appreciate the patience

**Comment by mrverdant13** [comment]
An update here
The fix has been included in coverde@0.2.0

**Comment by JCQuintas** [comment]
Nice, thanks for the fix. It'll take me some time to test this solution though, but I appreciate the work :)

I'm currently using a workaround, so this was not urgen, mostly an improvement 😅
